uMenu/include/ui/ui_QuickMenu.hpp

```diff
@@ -56,7 +64,12 @@ namespace ui {
             }
             
             inline void Toggle() {
-                this->on = !this->on;
+                if(ui::quickmenu_utils::quickMenuInputsToIgnore>0){
```

Comment:
I'd also make this function not inline anymore, since it's not a trivial thing anymore

Reply:
changing this :) , you're right